src/Cpanel/PluginActions.php

```diff
@@ -16,33 +16,39 @@ class PluginActions extends AbstractPluginActions
     protected $userConfig;
     protected $composer;
 
-    public static $CONFIG = array(
```

Comment:
> As of PHP 5.4 you can also use the short array syntax, which replaces array() with [].

from http://php.net/manual/en/language.types.array.php

In WordPress we support  PHP 5.3.10. We need to use `array()` rather than `[]`